apps/src/standaloneVideo/Video.tsx

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types';
+import React, {useRef, useLayoutEffect, useEffect, useState} from 'react';
+import styles from './video.module.scss';
+
+interface VideoProps {
+  children: React.ReactNode;
+  src: string | undefined;
+}
+
+function useWindowSize() {
+  const [size, setSize] = useState([
+    document.documentElement.clientWidth,
+    document.documentElement.clientHeight,
+  ]);
+  useLayoutEffect(() => {
+    function updateSize() {
+      const width = document.documentElement.clientWidth;
+      const height = document.documentElement.clientHeight;
+      setSize([width, height]);
+    }
+    window.addEventListener('resize', updateSize);
+    updateSize();
+    return () => window.removeEventListener('resize', updateSize);
+  }, []);
+  return size;
+}
+
+/**
+ * Renders a simple modal video player.
+ */
+
+const Video: React.FunctionComponent<VideoProps> = ({children, src}) => {
+  const childrenAreaHeight = 70;
+
+  let [targetWidth, targetHeight] = useWindowSize();
+  targetWidth -= 80;
+  targetHeight -= 100 + childrenAreaHeight;
```

Comment:
nit: constants and/or docs for some of these adjustment values?

Reply:
Good call; updated.